requirements.txt

```diff
@@ -1,21 +1,14 @@
-jsonschema
-markdown
-paste
-pylint
-pymongo
-pyopenssl
-python-dateutil
-pytz
-requests
-tzlocal
-webapp2
-webob
-rfc3987
-nose
-toml
-PasteScript
-pytest
-pytest-cov
-pytest-watch
-coverage
-coveralls
+# Production packages
```

Comment:
Removed a number of packages that appeared to no longer be used. Would appreciate @gsfr or @rentzso double-checking my work.